### PR TITLE
Cherry Pick PR to PROD-120

### PR DESCRIPTION
**Pull request by dedywahyudi**
Cherry Pick from these PR:

https://github.com/topcoder-platform/mfe-customer-work/pull/436/commits
https://github.com/topcoder-platform/mfe-customer-work/pull/435/commits

### CODE REVIEW

**Review by brooketopcoder — CHANGES_REQUESTED** [review]
thanks for knocking this out, @dedywahyudi !

we just need to narrow the scope of the changes to only the items that will be in the current release. 

https://topcoder.atlassian.net/browse/PROD-2144

The website design v2 and problem & data advisory changes are in the following release:

https://topcoder.atlassian.net/browse/PROD-2145

**Comment by dedywahyudi** [comment]
@brooketopcoder 

Can we ignore this PR. There are some conflicts that not merged correctly on this PR.
I created new PR by cherry pick from previous PR and all commits you made on "clone-PROD-120" branch: 
https://github.com/topcoder-platform/mfe-customer-work/pull/454

**Comment by dedywahyudi** [comment]
I'll reply all feedback above.
But, please check another PR for the fixes: https://github.com/topcoder-platform/mfe-customer-work/pull/454

File: src/routes/Products/components/BasicInfo/index.jsx [hunk 1]
```diff
@@ -44,27 +53,51 @@ const BasicInfo = ({
   bannerData,
   isLoggedIn,
 }) => {
-  const [formData, setFormData] = useState({
+  const defaultFormData = {
     projectTitle: { title: "Project Title", option: "", value: "" },
+    description: { title: "Description", option: "", value: "" },
```
[the remark this answers is not in the document]
Comment (dedywahyudi) [reply src/routes/Products/components/BasicInfo/index.jsx:58]: @brooketopcoder After new PR this should be fixed.

File: src/components/Banners/FeaturedWorkTypeBanner/styles.module.scss [hunk 2]
```diff
@@ -5,19 +5,21 @@
...
+  &.website-design,
+  &.data-exploration,
+  &.review--payment {
+    background: linear-gradient(84.92deg, #065D6E 2.08%, #06596E 2.09%, #3E3B91 97.43%), #FFFFFF;
   }
-  &.find-me-data {
-    background: linear-gradient(84.92deg, #723390 2.08%, #8C384F 97.43%);
+
+  &.find-me-data,
+  &.problem-statement--data-advisory,
```
[the remark this answers is not in the document]
Comment (dedywahyudi) [reply src/components/Banners/FeaturedWorkTypeBanner/styles.module.scss:20]: @brooketopcoder This should be fixed too

File: src/components/Banners/FeaturedWorkTypeBanner/styles.module.scss [hunk 3]
```diff
@@ -55,9 +61,24 @@
       background: url("../../../assets/images/data-exploration.png");
       background-size: cover;
     }
-    &.find-me-data {
+    &.find-me-data,
+    &.problem-statement--data-advisory,
```
[the remark this answers is not in the document]
Comment (dedywahyudi) [reply src/components/Banners/FeaturedWorkTypeBanner/styles.module.scss:65]: @brooketopcoder This should be fixed too

File: src/routes/Products/components/BasicInfo/index.jsx [hunk 4]
```diff
@@ -84,25 +129,38 @@ const BasicInfo = ({
...
-  const baseUrl = `/self-service/work/new/${isDataExploration ? "data-exploration" : "find-me-data"
-    }`;
+  let basePath;
+  if (isDataExploration) {
+    basePath = "data-exploration";
+  } else if (isFindMeData) {
+    basePath = "find-me-data";
+  } else if (isWebsiteDesign) {
+    basePath = "website-design";
+  } else if (isDataAdvisory) {
```
[the remark this answers is not in the document]
Comment (dedywahyudi) [reply src/routes/Products/components/BasicInfo/index.jsx:154]: @brooketopcoder Done, this is removed

File: src/routes/Products/components/BasicInfo/index.jsx [hunk 5]
```diff
@@ -123,13 +181,11 @@ const BasicInfo = ({
       dispatch(triggerAutoSave(true));
     }
 
-    if (
-      basicInfo && basicInfo?.projectTitle?.value.length > 0
-    ) {
+    if (basicInfo && basicInfo?.projectTitle?.value.length > 0) {
```
[the remark this answers is not in the document]
Comment (dedywahyudi) [reply src/routes/Products/components/BasicInfo/index.jsx:184]: @brooketopcoder cool, this is fixed now.

File: src/routes/Products/components/BasicInfoForm/index.jsx [hunk 6]
```diff
@@ -4,21 +4,35 @@
...
 import HelpBanner from "components/HelpBanner";
+import HelpIcon from "components/HelpIcon";
 import PageDivider from "components/PageDivider";
 import PageP from "components/PageElements/PageP";
 import PageRow from "components/PageElements/PageRow";
 import RadioButton from "components/RadioButton";
 import FormInputTextArea from "components/FormElements/FormInputTextArea";
 import ServicePrice from "components/ServicePrice";
+// TODO: Move this component to /components
+import ColorOptions from "../../../Branding/components/ColorOptions";
```
[the remark this answers is not in the document]
Comment (dedywahyudi) [reply src/routes/Products/components/BasicInfoForm/index.jsx:17]: @brooketopcoder ok, i think this is fixed too

**Pull request closed without merging**